### ISSUE TITLE
NLog.config file not added to project when installing NuGet Package using PackageReference

### ISSUE THREAD

**Issue by shaggygi** [NLog/NLog#2757]
**NLog version**: (4.5.6)

**Platform**: .NET 4.6.1

Referencing previous issue https://github.com/NLog/NLog/issues/2461#issuecomment-394013813

Steps to reproduce...

1. File > New > Project > WPF App
2. Right-click References > Manage NuGet Packages...
3. Select Browse tab > type NLog.Config > Install

NLog.config file should be added to project, but don't see like I did before starting to use the new PackageReference for NuGet.

I can see the file in the Debug folder.

**Comment by 304NotModified** [comment]
What's the NLog version?

**Comment by shaggygi** [comment]
NLog and NLog.Config version: 4.5.6

**Comment by 304NotModified** [comment]
what's your visual studio version?

**Comment by shaggygi** [comment]
VS 2017 15.7 RTM

**Comment by ggirard07** [comment]
Same here with NLog.Config 4.5.7.
I am not sure what is accomplished by the package is actually supported in nuget package reference mode...

**Comment by 304NotModified** [comment]
it should work on both modes. This has been fixed in the past, not sure if it's the same problem

**Comment by ggirard07** [comment]
In my case I don't have any file coming from `content` in the packages (nlog.config, nlog.xsd).
Also I noticed `nlog.config` is using an `Install.ps1` script, which is no longer supported in package reference mode (https://docs.microsoft.com/en-us/nuget/reference/migrate-packages-config-to-package-reference#package-compatibility-issues)

**Comment by 304NotModified** [comment]
Thanks for reporting. 

Closing this in favor of https://github.com/NLog/NLog/issues/2823
